pandas/core/base.py

```diff
@@ -472,7 +472,7 @@ def is_any_frame() -> bool:
             except ValueError:
 
                 # we have a dict of scalars
-                result = Series(result, name=getattr(self, "name", None))
+                result = Series(result)
```

Comment:
That will probably break a lot of tests. Could you catch the case, when ``getattr(self, "name", None)`` is not hashable and keep the existing logic in all other cases? Also please add tests